libtransmission/log.cc

```diff
@@ -111,7 +111,7 @@ void logAddImpl(
     {
         auto* const newmsg = new tr_log_message{};
         newmsg->level = level;
-        newmsg->when = tr_time();
+        newmsg->when = std::chrono::system_clock::now();
```

Comment:
`tr_time()` is meant to be "inexpensive". I guess logging in hot paths should generally be avoided so this isn't an issue, but maybe it is?

Reply:
Hmm... How else do you suggest we can implement this if we want sub-second precision?

In the case of the daemon, it has been calling `std::chrono::system_clock::now()` via `tr_logGetTimeStr()` for every log message, so there isn't any extra overhead added in this PR.

For other clients, the overhead for logging is raised to around the same level as the daemon.

Reply:
I was simply asking if this might affect performance. If it doesn't then there's nothing to suggest. I'm not even sure we need millisecond precision in the other place down below (genuinely wondering if it's useful) but if performance isn't an issue then I don't care either way. As you hopefully understand, wall clock may drift so it's not exactly reliable for any sort of measurements.